src/helpers/helpers.ts

```diff
@@ -45,7 +45,7 @@ export async function copyFileOrDir(
       if (error) {
         reject(error);
       }
-      resolve();
+      resolve(null);
```

Comment:
Why?

```suggestion
      resolve();
```

Reply:
`resolve();` results in the following error:
src/helpers/helpers.ts(48,7): error TS2794: Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?

Reply:
> `resolve();` results in the following error:
> src/helpers/helpers.ts(48,7): error TS2794: Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?

@sorhtyre neat, this is something newly-caught by fresh new TypeScript 4.1. Here the compiler is shouting at us that we said we'd return/resolve something (of type `any`), but we're returning nothing (of type `void`).

So, rather than return `null` (which is useless to a caller, our function either succeeds at copying a file, or it throws, but it never has anything to return), the right thing to do is to adjust the return type of our function (at line 42) to return a `Promise<void>` instead of a `Promise<any>`. Then the current `resolve()` will work. Give it a try.

Reply:
That is correct. But I wasn't sure if you wanted to change the type. I'll make the change and test it.